src/backend/networking/peloton_service.cpp

```diff
@@ -267,12 +273,31 @@ void PelotonService::Heartbeat(::google::protobuf::RpcController* controller,
       LOG_ERROR("No response: site is null");
     }
 
+
+    /*
```

Comment:
Do not use /* .. */ style comments. Use just // comments even for multi-line comments. Later, if we need to comment something quickly for temporary hacking, then we can use /* .. */ style comments.